lib/PlaidClient.js

```diff
@@ -285,6 +285,7 @@ function(asset_report_token, cb) {
     body: {
       asset_report_token: asset_report_token,
     },
+    binary: true,
```

Comment:
I am guessing that the current behavior is that we return a base64 encoded string, is that right @byF ? You could convert this back into a buffer if you needed to, no? 

I'm a bit wary of this change, since it is a breaking change in the clien'ts API - assuming the current implementation does work in the way I posited above. Perhaps I am missing something, though.

Reply:
Nope, it's not that case unfortunately (or I missed something). The request lib was encoding the binary data as utf8 string which made them unusable.

I didn't figure out how to work around it (tried various Buffer.from variations, but nothing produced correct binary data), that's why I digged through the library to fix it :)

Reply:
Also regarding breaking change - bindings say it returns `Buffer`, so anyone who depends on it knows they work around a bug (I don't know how though, I'd love to learn that)